packages/fuel-indexer-tests/src/fixtures.rs

```diff
@@ -527,46 +526,49 @@ pub mod test_web {
         HttpResponse::Ok()
     }
 
-    async fn fuel_indexer_test_transferout(
-        state: web::Data<Arc<AppState>>,
-    ) -> impl Responder {
-        let call_params =
-            CallParameters::new(1_000_000, fuels::types::AssetId::default(), 1000);
-
-        let _ = state
-            .contract
-            .methods()
-            .trigger_transferout()
-            .append_variable_outputs(1)
-            .tx_params(tx_params())
-            .call_params(call_params)
-            .expect("Could not set call parameters for contract method")
-            .call()
-            .await;
-
-        HttpResponse::Ok()
-    }
-
-    async fn fuel_indexer_test_messageout(
-        state: web::Data<Arc<AppState>>,
-    ) -> impl Responder {
-        let call_params =
-            CallParameters::new(1_000_000, fuels::types::AssetId::default(), 1000);
-
-        let _ = state
-            .contract
-            .methods()
-            .trigger_messageout()
-            .append_message_outputs(1)
-            .tx_params(tx_params())
-            .call_params(call_params)
-            .expect("Could not set call parameters for contract method")
-            .call()
-            .await
-            .unwrap();
-
-        HttpResponse::Ok()
-    }
+    // TODO: TransferOut and MessageOut tests are currently ignored
```

Comment:
Why are these tests flaky?

Reply:
There seems to have been a change made between `v0.40` and `v0.45` regarding how to get variable outputs, which are necessary for the `*Out` receipt tests. I'm already making changes to the E2E tests as part of #1194, so I'll fix and re-enable them as part of that work.